server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/DeletedMessagesVaultTest.java

```diff
@@ -467,7 +468,7 @@ public void vaultExportShouldExportZipContainsVaultMessagesToShareeWhenJmapDelet
         String fileLocation = exportAndGetFileLocationFromLastMail(EXPORT_ALL_HOMER_MESSAGES_TO_BART, bartAccessToken);
 
         try (ZipAssert zipAssert = assertThatZip(fileSystem.getResource(fileLocation))) {
-            zipAssert.containsOnlyEntriesMatching(hasName(messageIdOfHomer));
+            zipAssert.containsOnlyEntriesMatching(hasName(messageIdOfHomer + EML_EXTENSION));
```

Comment:
I'm a bit late on that topic but this `hasName` doesn't respect the usual fluent design of assertj assertions.
In this case, it shoud be :
```java
zipAssert.allSatisfies(entry -> assertThat(entry).hasName("foo"))
```